plugins/x86/x86_lifter.ml

```diff
@@ -481,7 +481,8 @@ module ToIR = struct
                 iv := int @@ Word.of_int ~width:byte i;
                 tmp_dst :=
                   var tmp_dst lor
-                  ((cast unsigned t_width (var elt)) lsl (var iv * int bits))
+                  ((cast unsigned t_width (var elt))
```

Comment:
the changes to this file are not necessary, could you please discard them?

Reply:
👍